File: Beecrowd_Parte3/OndeEstaMinhaCaneta_2495.c
#include <stdio.h>

int main() {
    int N;
    
    while (scanf("%d", &N) != EOF) {
        long long sumTotal = (long long) N * (N + 1) / 2;
        long long sumReturned = 0;
        
        for (int i = 0; i < N - 1; i++) {
            int returnedPen;
            scanf("%d", &returnedPen);
            sumReturned += returnedPen;
        }
        
        long long missingPen = sumTotal - sumReturned;
        printf("%lld\n", missingPen);
    }
    
    return 0;
}

File: Beecrowd_Parte3/OndeEstaMinhaCaneta_2495.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        while (sc.hasNext()) {
            int N = sc.nextInt();
            long sumTotal = (long) N * (N + 1) / 2;
            long sumReturned = 0;
            
            for (int i = 0; i < N - 1; i++) {
                int returnedPen = sc.nextInt();
                sumReturned += returnedPen;
            }
            
            long missingPen = sumTotal - sumReturned;
            System.out.println(missingPen);
        }
        
        sc.close();
    }
}
